BILLthebuilder/automart
# #166397764 Create signup endpoint

* It creates an endpoint for a user to signup
#### Description of Task to be completed?
* The `POST/api/v1/auth/signup` endpoint is created here
#### How should this be manually tested?
* Clone the repository
* Checkout to the `ft-user-register-166397764` branch
* Clone the repo
* Run `npm install` to install package dependencies
* Run `npm test` to run tests
* All the tests should pass
* Run `npm start`
* Test the `/api/v1/auth/signup` request endpoint on postman
* It should return a `status` of `201`
* Errors are handled properly to prevent app crashing
#### What are the relevant pivotal tracker stories?
* [#166397764](https://www.pivotaltracker.com/story/show/166397764)
#### Screenshots
![Screenshot from 2019-06-06 13-23-55](https://user-images.githubusercontent.com/24655101/59026429-080f4580-885f-11e9-834c-53faff77a145.png)
![Screenshot from 2019-06-06 13-10-36](https://user-images.githubusercontent.com/24655101/59026295-ba92d880-885e-11e9-81c1-c143c682c8e8.png)
